modules/vault/outputs.tf

```diff
@@ -19,3 +19,17 @@ output "vault_unseal_bucket" {
 output "vault_dynamodb_table" {
     value = aws_dynamodb_table.vault-dynamodb-table.id
 }
+
+// ----------------------------------------------------------------------------
+// The Vault user id if one got created
+// ----------------------------------------------------------------------------
+output "vault_user_id" {
+  value =  var.vault_user == "" ? aws_iam_access_key.jenkins-x-vault[0].id : ""
+}
+
+// ----------------------------------------------------------------------------
+// The Vault user secret if one got created
+// ----------------------------------------------------------------------------
+output "vault_user_secret" {
+  value =  var.vault_user == "" ? aws_iam_access_key.jenkins-x-vault[0].secret : ""
+}
```

Comment:
I'm not sure about creating the secret key in Terraform. Do we really want to output these values?

Reply:
How else do you want the user to get to it? How would your workflow? You can always opt-out and specify the vault user directly. It is also the reason why you should always keep Terraform state encrypted in a backend. It can contain sensitive data.

Reply:
@ccojocar, what do you think? The context is around the additionally required AWS IAM user for Vault when using Terraform to create all required resources for Jenkins X on an EKS cluster. 

The change here is supposed to make things easier for users to quickly get up and running and to reduce the number of manual steps. For that reason, this script will create an IAM user and credentials if permissions allow doing so. If you explicitly specify an IAM user name as variable the user won't get created and you are responsible yourself to create the access key. 

WDYT?